top-car-trading-platform-chooses-scale-out-database-as-mysql-alternative.md

```diff
@@ -0,0 +1,307 @@
+---
+title: Top Car Trading Platform Chooses a Scale-out Database as MySQL Alternative
+author: ["Chehaoduo"]
+date: 2021-01-12
+summary: Chehaoduo, the largest auto trading platform in China, migrated from MySQL to TiDB and got rid of the troublesome process of MySQL sharding and schema changes.
+image: /images/blog/chehaoduo-mysql-alternative-scale-out-database.png
+url: /success-stories/top-car-trading-platform-chooses-scale-out-database-as-mysql-alternative/
+customer: Chehaoduo
+customerCategory: Internet
+logo: /images/blog/customers/chehaoduo-logo.png
+---
+
+**Industry:** Automobile
+
+**Author:** Kevin Ge, Jiansheng Han, and Sheyang Wang (TiDB virtual team at Chehaoduo)
+
+![Chehaoduo banner](media/chehaoduo-mysql-alternative-scale-out-database.png)
+
+[Chehaoduo](https://www.crunchbase.com/organization/guazi-com) is an online trading platform for both new cars and personal used cars. Founded in 2015, it is now the largest auto trading platform in China, valued at $9 billion in its series D round of funding last year.
+
+In the early stages of Chehaoduo, to quickly adapt to our application development, we chose MySQL as our major database. However, as our business evolved, we were greatly troubled by the complication of MySQL sharding and schema changes. In face of this dilemma, we found an alternative database to MySQL: [TiDB](https://docs.pingcap.com/tidb/stable), an open-source, MySQL compatible database that scales out to hold massive data.
+
+In this post, I’ll share with you why we chose TiDB and how it empowers our application to provide better service for our customers.
```

Comment:
Please update all quotes to straight quotes.

Reply:
fixed